### ISSUE TITLE
Inconsistency between "blocked" and "Likely blocked" communication of results

### ISSUE THREAD

**Issue by agrabeli** [ooni/probe#1942]
During the OONI Partner Training, a participant mentioned that it's confusing that it says "Blocked" in the OONI Probe test results overview, but when they access the individual measurement, it says "Likely blocked". 

We should make the communication of results consistent, to avoid creating confusion. My understanding is that we went with "Blocked" in the test results overview due to space issues, but perhaps there's a workaround?

**Comment by hellais** [comment]
Yes, I don't think "Likely blocked" is going to fit inside of the result overview.
